sensor_msgs/PointCloud2 point_cloud
---
geometry_msgs/Point grasp_point_1
geometry_msgs/Point grasp_point_2
